### DISCUSSION TITLE
Is AEM 6.5 or any version is supported by Skywalking java agent. If so how we can configure application for monitoring

### DISCUSSION THREAD

**Discussion by the-rough-page** [apache/skywalking#8918] [Q&A] [answered]
Is there any version of AEM skywalking does support it?
If so how can we configure the application for monitoring

**Comment by Superskyyy** [comment, marked as answer]
No, it is not supported, if the documentation does not say supported, it is by default not supported officially. 

Although we welcome the plugin contribution.

**Comment by the-rough-page** [comment]
one more thing application is written in Java and instead of Apache using AEM so do we need to develop the plugin for that or in any configuration change will help to do it

**Reply by Superskyyy** [reply]
Sorry I don't understand the problem situation, what do you mean by "written in Java, instead of Apache using AEM"? Could you elaborate on it. 

Actually I think you may have some misunderstandings of how apm agent works, have you read the instrumentation docs? It is not magic.
